File: test/fixtures/php/get_with_env_var.php
<?php
$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, 'http://localhost:28139/v2/images?type=distribution');
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'GET');
curl_setopt($ch, CURLOPT_HTTPHEADER, [
    'Content-Type: application/json',
    'Authorization: Bearer ' . getenv('DO_API_TOKEN') ?? '',
]);

$response = curl_exec($ch);

curl_close($ch);

File: test/fixtures/kotlin/get_with_env_var.kt
import java.io.IOException
import java.lang.System
import okhttp3.OkHttpClient
import okhttp3.Request

val client = OkHttpClient()

val request = Request.Builder()
  .url("http://localhost:28139/v2/images?type=distribution")
  .header("Content-Type", "application/json")
  .header("Authorization", "Bearer " + System.getenv("DO_API_TOKEN") ?: "")
  .build()

client.newCall(request).execute().use { response ->
  if (!response.isSuccessful) throw IOException("Unexpected code $response")
  response.body!!.string()
}
